doc/release/upcoming_changes/14777.compatibility.rst

```diff
@@ -0,0 +1,7 @@
+Changed random variate stream from `numpy.random.Generator.integers`
+--------------------------------------------------------------------
+When the argument ``dtype`` is used to specify an 8 or 16 bit data type in
+the function `numpy.random.Generator.integers`, the stream of variates that
+is generated will not be the same as in the previous release.  There was a
```

Comment:
It would read better to put the last sentence first, something like.
```
There was a bug in `numpy.random.Generator.integers` that caused biased sampling of 8 and 16 bit integer types. Fixing that bug has changed the output stream from what is was in previous releases.
```

Reply:
@charris, yes, that sounds better.  I pushed an update.

Reply:
@charris, yes, that sounds better.  I pushed an update.